plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2877,8 +2886,13 @@ public void testSplitPruningForFilterOnNonPartitionColumn(DataMappingTestSetup t
             verifySplitCount("SELECT row_id FROM " + tableName, 2);
             verifySplitCount("SELECT row_id FROM " + tableName + " WHERE col = " + sampleValue, expectedSplitCount);
             verifySplitCount("SELECT row_id FROM " + tableName + " WHERE col = " + highValue, expectedSplitCount);
-            verifySplitCount("SELECT row_id FROM " + tableName + " WHERE col > " + sampleValue, expectedSplitCount);
-            verifySplitCount("SELECT row_id FROM " + tableName + " WHERE col < " + highValue, expectedSplitCount);
+
+            // ORC max timestamp statistic is truncated and rounded to the nearest millisecond.
```

Comment:
> truncated and rounded

truncated (aka rounded down) or rounded?

let's make it clear

Reply:
clarified both. thanks!